nupic/embodied/utils/model_parts.py

```diff
@@ -0,0 +1,243 @@
+import torch
+import numpy as np
+
+
+def flatten_dims(x, dim):
+    """Flatten dimension dim of x.
```

Comment:
I love numpy style docstrings, they are very clean. But in nupic we use the more traditional sphinx style docstring. At some point would be good to standardize, but it doesn't really matter now if we have mixed types of docstrings for a while.